host-controller/src/main/java/org/jboss/as/domain/controller/operations/DomainServerLifecycleHandlers.java

```diff
@@ -391,4 +396,27 @@ public void execute(OperationContext context, ModelNode operation) throws Operat
         }
     }
 
+    public static void registerServerLifeCycleOperationsTransformers(ResourceTransformationDescriptionBuilder builder) {
+        builder.addOperationTransformationOverride(SUSPEND_SERVERS).setReject().end()
+                .addOperationTransformationOverride(RESUME_SERVERS).setReject().end()
+                .addOperationTransformationOverride(STOP_SERVERS)
+                .setCustomOperationTransformer((context, address, operation) -> {
+                    if (operation.hasDefined(ModelDescriptionConstants.TIMEOUT)) { //timeout is new parameter, reject if present, otherwise it is fine
```

Comment:
You can use the builder for this as well

    .addRejectCheck() 

It works similar to for attributes, but called on the OperationTransformationOverrideBuilder applies to operation parameters.